cmd/singularity/pull_test.go

```diff
@@ -56,16 +50,20 @@ func imagePull(t *testing.T, library, pullDir string, imagePath string, sourceSp
 	return cmd.CombinedOutput()
 }
 
+// tmpDirReturn will return a tmp dir path in /tmp.
+func tmpDirReturn(t *testing.T) string {
```

Comment:
Can we change this to `makeTmpDir` or something to that effect? Having `Return` in the name isn't very descriptive.

Reply:
Actually, this should not even be in this PR, I'll make a new PR that improves the `pull_test.go`

Reply:
Nevermined, these changes need to be in this PR (this PR changes the exit code).